utils/notification_service.py

```diff
@@ -49,8 +50,8 @@
     "PyTorch pipelines": "torch_pipeline",
     "TensorFlow pipelines": "tf_pipeline",
     "Examples directory": "example",
-    "DeepSpeed": "deepspeed",
     "Quantization": "quantization",
+    "DeepSpeed": "deepspeed",
```

Comment:
not sure if the switch of order is intended.

Reply:
not intended !